texi2rst.py

```diff
@@ -90,6 +91,9 @@ def from_xml_string(xml_src):
     xml_src = xml_src.replace('&textlsquo;', "'")
     xml_src = xml_src.replace('&textrsquo;', "'")
     xml_src = xml_src.replace('&eosquest;', "?")
+    xml_src = xml_src.replace('&expansion;', '→')
+    xml_src = xml_src.replace('&result;', '⇒')
+    xml_src = xml_src.replace('&errorglyph;', 'error')
```

Comment:
What is `&errorglyph;`?  Where does it come from?  If it's some kind of toolchain error that's showing up in a conversion, should we fail more forcefully if it occurs?

Where do `&expansion;` and `&result;` come from? (this is more out of curiosity)

Reply:
> What is `&errorglyph;`? Where does it come from? If it's some kind of toolchain error that's showing up in a conversion, should we fail more forcefully if it occurs?

It's used in an example: https://gcc.gnu.org/onlinedocs/cpp.pdf (page 21), see `error` in a rectangle.

Reply:
> Where do `&expansion;` and `&result;` come from? (this is more out of curiosity)

First, one is used also in CPP manual and the second one in CPP internals manual.

```
INC(foo);
7→ ++foo;
```

and

```
baz
⇒
foo bar baz
```

Reply:
Thanks!  (I couldn't see the "error" on page 21 of the document, but then realized it's on the 21st page of the document, which has "16" as the given page number in the corner.  Gahhh).

Looks reasonable.